ergonomics/ergonomics.md

```diff
@@ -36,18 +36,33 @@ Software engineers often spend 4 to 10 hours a day at their computers. It's vita
 - Not putting off bathroom use
 
 ### Supplemental Materials
-
+- [Ergonomics (video walkthrough of slides)](https://drive.google.com/open?id=1s2yDFOSpTiiw_75aF6DjyMRtdK9EuyaD)
+- [Ergonomics (slides)](https://docs.google.com/presentation/d/1q-X4-bPDdyk8Hr3ne-Rr0SMFMP7YRdJQz9AzAZ2fR1A/edit?usp=sharing)
 - [7 Ways to Be A Healthier Programmer](https://successfulsoftware.net/2008/10/26/7-ways-to-be-a-healthier-programmer/)
 - [10 Exercises for Software Engineers](https://www.linkedin.com/pulse/10-must-do-exercises-ergonomics-software-developers-designers-wang/)
 - [CalOSHA's "Easy Ergonomics for Desktop Users"](https://www.dir.ca.gov/dosh/dosh_publications/ComputerErgo.pdf)
 
-
 ### Lesson
-
-[Ergonomics (video walkthrough of slides)](https://drive.google.com/open?id=1s2yDFOSpTiiw_75aF6DjyMRtdK9EuyaD)
-
-[Ergonomics (slides)](https://docs.google.com/presentation/d/1q-X4-bPDdyk8Hr3ne-Rr0SMFMP7YRdJQz9AzAZ2fR1A/edit?usp=sharing)
-
+- What is Ergonomics?
+  - "Ergonomics is an applied science concerned with designing and arranging things people use so that the people and things interact most efficiently and safely".
+  
+- Points to Remember
+  - Do Not Sit Still!
+    - Sitting still for prolonged is not healthy ,it causes decline in circulation of blood and oxygen and muscles aches in neck, shoulders,back,legs.
+    - It causes eye strain when working on computer.
+    - Be kind to your bladder, Do not force yourself to keep working if you need to use bathroom.
+    
+  - Take frequent breaks
+    - Try to work for 50-55 minutes of every hour  and use last 5-10 minutes to stretch your body and walk around.
+  
+  - Neutral body positioning
+    - Comfortable working posture in which your joints are naturally aligned,Elbows are bent 90 degrees & forearms parallel with floor ,Knees & hips bent 90 degrees & thighs parallel with floor,Computer monitor is at eye-level, not above or below.
```

Comment:
No capital letters in the middle of the sentence please. If you are using capital letters, please use full stop instead.

Reply:
done